ssh/ssh_tunnel.py

```diff
@@ -54,7 +54,7 @@ def __init__(self, ssh_user, ssh_key_path, host, port=22):
         check_call(start_tunnel)
         logger.debug('SSH Tunnel established!')
 
-    def remote_cmd(self, cmd, timeout=None):
+    def remote_cmd(self, cmd, raw_output=False, timeout=None):
```

Comment:
why do we ever want to run without raw output mode? It seems like generally the better option for running the commands